core/src/test/java/org/elasticsearch/test/ExternalNode.java

```diff
@@ -199,6 +212,7 @@ synchronized Client getClient() {
             Settings clientSettings = settingsBuilder().put(externalNodeSettings)
                     .put("client.transport.nodes_sampler_interval", "1s")
                     .put("name", "transport_client_" + nodeInfo.getNode().name())
+                    .put("path.home", PathUtils.get(".").toAbsolutePath())
```

Comment:
Its the CWD of the external node which I picked because it worked but without any strong reasoning. If you have a better idea I'll certainly use it!